misc/python/materialize/feature_benchmark/executor.py

```diff
@@ -67,6 +70,9 @@ def Kgen(self, topic: str, args: List[str]) -> Any:
             "kgen", f"--topic=testdrive-{topic}-{self._seed}", *args
         )
 
+    def DockerMem(self) -> int:
+        return self._composition.mem("feature-benchmark-materialized-1")
```

Comment:
This probably doesn't work when used with `bin/mzcompose --project-name`. Not sure if it's easy to find the project name from here though.